easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-gmvolf-17.02.eb

```diff
@@ -0,0 +1,26 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.22'
+downloadversion = version.replace('.','')
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'gmvolf', 'version': '17.02'}
```

Comment:
As discussed, please use either `GCC` or `dummy` for this PR

Reply:
Updated recipes with `dummy` toolchain